schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,275 @@
+                the user is expected to translate appropriately to match with an element value below.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="None"/>
+            <xs:enumeration value="Low"/>
+            <xs:enumeration value="Medium"/>
+            <xs:enumeration value="High"/>
+            <xs:enumeration value="Critical"/>
+            <xs:enumeration value="Unknown"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:simpleType name="scoreValueType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Numerical representation of the vulnerability score.
+                Must be a number between 0 - 10 (maps to lowest severity - highest severity)
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:decimal">
+            <xs:minInclusive value="0"/>
+            <xs:fractionDigits value="1"/>
+            <xs:maxInclusive value="10"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:simpleType name="scoreSourceType" final="restriction">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+                Specifies the risk scoring methodology/standard used.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="CVSSv2">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        The rating is based on CVSS v2 standard
+                        https://www.first.org/cvss/v2/guide
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="CVSSv3">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        The rating is based on CVSS v3 standard
+                        https://www.first.org/cvss/v3.1/specification-document
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="OWASP Risk">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        The rating is based on OWASP Risk Rating
+                        https://www.owasp.org/index.php/OWASP_Risk_Rating_Methodology
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="Open FAIR">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        The rating is based on Open FAIR specification
+                        http://www.opengroup.org/subjectareas/security/risk
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="PCI DSS">
```

Comment:
I looked at the PCI DSS doc and it doesn't specifically state a method for evaluating risk. Rather, it provides an example of how to do so. The only requirement appears to be that it's done. But I'm unclear if PCI DSS should actually be a valid risk rating method or not. Comments?

Reply:
> I looked at the PCI DSS doc and it doesn't specifically state a method for evaluating risk. Rather, it provides an example of how to do so. The only requirement appears to be that it's done. But I'm unclear if PCI DSS should actually be a valid risk rating method or not. Comments?

Think you are right. 👍  This came as an option in our initial research. But based on above and internal discussion this doesn't look like it can be placed as a risk rating method.   b2e2b00